wscript

```diff
@@ -151,10 +151,10 @@ def configure(conf):
 
 
 	# test for GStreamer libraries
-
-	gst_version_str = conf.check_cfg(package = 'gstreamer-1.0 >= 1.2.0', modversion = "gstreamer-1.0", uselib_store = 'GSTREAMER', args = '--cflags --libs', mandatory = 1)
+	conf.check_cfg(package = 'gstreamer-1.0 >= 1.2.0', uselib_store = 'GSTREAMER', args = '--cflags --libs', mandatory = 1)
+	gst_version_str = conf.check_cfg(modversion = "gstreamer-1.0", uselib_store = 'GSTREAMER', args = '--cflags --libs', mandatory = 1)
 	gst_version = [int(x) for x in re.match('(\d*)\.(\d*)\.(\d*)', gst_version_str).groups()]
-	conf.env['GSTREAMER_VERSION'] = gst_version
+	print("gst_version %s" %(gst_version))
```

Comment:
Can you use the log functions from the waf Logs module instead?

Reply:
Actually I added the print for me to check that I actually get the correct gstreamer version.
Then I deleted the assignment into the conf.env space instead of deleting the debug code.

I update the pull request to keep the removed line but delete the added print..

Reply:
@dv1 

Is there anything else which prevents this one from being merged?
meta-freescale with current master oe-core does currently fail because of the need for python2.

(Along with the dependencies libimxvpuapi and libimxdmabuffer which both have similar merge requests.)

Reply:
Just time ... have lots of stuff to do :/

Anyway, looks fine, I'll merge it.